app/src/main/res/layout/guide_filter_popup.xml

```diff
@@ -10,7 +9,7 @@
         android:layout_alignParentTop="true"
         android:layout_centerHorizontal="true"
         android:textSize="30sp"
-        android:textColor="@android:color/holo_blue_light"
+        android:textColor="?attr/defaultBrandColor"
```

Comment:
Maybe a different label to avoid the word brand?

Reply:
I blame Google for this one. It comes from the base Android theme we're extending. :joy: